clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -864,7 +870,7 @@ std::string PointerVariableConstraint::mkString(Constraints &CS,
   }
 
   // TODO Remove comparison to RETVAR.
-  if (getName() == RETVAR && !ForItype)
+  if (UseName == RETVAR && !ForItype)
```

Comment:
Notice the TODO above; is the concern that it conveys exacerbated by allowing the choice of name to come in from the outside?

Reply:
fixed with John's suggestion above